src/pages/waiting-room-to-car/cat-c/components/vehicle-checks-modal/vehicle-checks-modal.cat-c.actions.ts

```diff
@@ -0,0 +1,10 @@
+import { Action } from '@ngrx/store';
+
+export const VEHICLE_CHECKS_VIEW_DID_ENTER = '[VehicleChecks] [Cat C] Vehicle Checks Did Enter';
```

Comment:
Why can't we reuse B+E here? This should be generic as analytics can handle it's own category setting

Reply:
yeah, extracting analytics effects and actions to common components

Reply:
ahh I was pretty sure that I tried to do this, but I didn't. It would mean that actions needs to be common as well and they are not at the moment. Let's come back to this later on